app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb

```diff
@@ -6,12 +6,13 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Credential < Mana
   alias_attribute :manager_id, :resource_id
   alias_attribute :manager, :resource
 
-  include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Credential
-  include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::TowerApi
-
   FRIENDLY_NAME = "Ansible Automation Inside Credential".freeze
 
   def self.provider_params(params)
     super.merge(:organization => ManageIQ::Providers::EmbeddedAnsible::AutomationManager.first.provider.default_organization)
   end
+
+  def self.notify_on_provider_interaction?
```

Comment:
Note:  Put this in since it was something that was also done for the previous version via the shared tower code.

Reply:
:+1: Sounds good...I purposely didn't copy that one over because I was doing it piecemeal and the first 2 classes I worked on didn't need it, but I agree it should come over.